Scripts/Runtime/WitRequestFactory.cs

```diff
@@ -35,9 +42,14 @@ public static WitRequest MessageRequest(this WitConfiguration config, string que
         /// <param name="config"></param>
         /// <param name="maxBestIntents"></param>
         /// <returns></returns>
-        public static WitRequest SpeechRequest(this WitConfiguration config, int maxBestIntents = 1)
+        public static WitRequest SpeechRequest(this WitConfiguration config, WitRequestOptions requestOptions, int maxBestIntents = 1)
```

Comment:
let's move `maxBestIntents` to `WitRequestOptions`

Reply:
^

Reply:
done.